opengrok-indexer/src/main/java/org/opengrok/indexer/history/Repository.java

```diff
@@ -548,6 +548,16 @@ static Boolean checkCmd(String... args) {
         return exec.exec(false) == 0;
     }
 
+    protected static String getCommand(String className, String propertyKey, String fallbackCommand) {
```

Comment:
I wouldn't mind a Class parameter instead of string. @vladak ?

Reply:
I concur.